src/components/Pupils/QuizEngineProvider/useQuizEngineContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from "react";
```

Comment:
looks like you've copied this redundant wrapper from somewhere else or AI wrote it for you. It's redundant and conflates hooks with context. Remove pls.